freeipa.spec.in

```diff
@@ -147,6 +147,7 @@ BuildRequires:  python-sssdconfig
 BuildRequires:  python-nose
 BuildRequires:  python-paste
 BuildRequires:  systemd-python
+BuildRequires:  python2-jinja2
```

Comment:
Good point. I also realized that I had the jinja2 dependency on the ipalib package, but it's actually the ipaclient package that needs it now. So I added it to both the python2 and python3 ipaclient packages.